### PR TITLE
refactor(output): remove `config` dependency

### PR DESCRIPTION
**Pull request by villevsv-upcloud**
Based on https://github.com/UpCloudLtd/upcloud-cli/pull/170

### CODE REVIEW

**Review by peknur — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+- Refactor output rendering. This removes `config` dependency from `output` package.
```
Comment (kangasta) [inline CHANGELOG.md:11]: I have omitted CHANGELOG entries in this project, if there are no end-user facing changes, but no strong opinion if this kind of developer facing changes should be included in the CHANGELOG.

File: internal/commands/runcommand.go [hunk 2]
```diff
@@ -153,19 +139,28 @@ func execute(command Command, executor Executor, args []string, parallelRuns int
 					returnChan <- executeResult{Job: index, Error: fmt.Errorf("cannot resolve argument: %w", argument.Error)}
 				} else {
 					// otherwise, execute and return results
-					executor.Debug("worker starting", "worker", index, "argument", argument.Resolved)
+					executor.Debug("worker stconfigarting", "worker", index, "argument", argument.Resolved)
```
Comment (kangasta) [inline internal/commands/runcommand.go:142]: Extra `config` in the middle of `starting`?

**Comment by kangasta** [comment]
Rebases this on top of `main`

**Comment by villevsv-upcloud** [comment]
Closing in favor of https://github.com/UpCloudLtd/upcloud-cli/pull/180

**Pull request closed without merging**